src/parser/class.js

```diff
@@ -122,6 +122,7 @@ module.exports = {
       } else if (
         allow_variables &&
         (this.token === this.tok.T_VARIABLE ||
+          this.token === this.tok.T_READ_ONLY ||
```

Comment:
I think we should add a version check here, such that parsing read only properties only works if `version >= 810`